patches/chrome-android-java-src-org-chromium-chrome-browser-suggestions-tile-SuggestionsTileView.java.patch

```diff
@@ -1,13 +1,13 @@
 diff --git a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/SuggestionsTileView.java b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/SuggestionsTileView.java
-index 219aad30b54a4cdbd9530b7f908ec07c4910b174..6ae04ddfe9e0188c81ba2ca3b81f568086299135 100644
+index 219aad30b54a4cdbd9530b7f908ec07c4910b174..032f16f18f71ad72eebec2c0e8bdae869d5b73a6 100644
 --- a/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/SuggestionsTileView.java
 +++ b/chrome/android/java/src/org/chromium/chrome/browser/suggestions/tile/SuggestionsTileView.java
 @@ -17,7 +17,7 @@ import org.chromium.url.GURL;
   * The view for a site suggestion tile. Displays the title of the site beneath a large icon. If a
   * large icon isn't available, displays a rounded rectangle with a single letter in its place.
   */
 -public class SuggestionsTileView extends TileView {
-+public class SuggestionsTileView extends org.chromium.chrome.browser.widget.tile.BraveTileWithTextView {
++public class SuggestionsTileView extends org.chromium.chrome.browser.suggestions.tile.BraveTileView {
```

Comment:
we can change the superclass with asm

Reply:
Addressed here https://github.com/brave/brave-core/commit/c3c611b45d6b7266a100913648a1c5530890ecc0